### ISSUE TITLE
Use of dash in properties conflicts with Javascript and Python usage

### ISSUE THREAD

**Issue by PaulWay** [RedHatInsights/insights-host-inventory#43]
After @jhjaggars's work breaking the `canonical_facts` JSON into separate JSON properties, e.g. `insights-id` and `fqdn`, some of these properties contain dashes.  As such they then cannot be imported into Javascript and used as object properties, because object property names cannot contain dashes.  For the same reason, they are not compatible with Django-rest-framework serializers as serializer fields are implemented as class methods which have the same restriction on using dashes.

I would recommend these be changed to underscores throughout.

This would also make them more consistent with properties like `display_name`, `created_on` and `modified_on`.

**Comment by PaulWay** [comment]
I think https://github.com/RedHatInsights/insights-host-inventory/pull/44 fixes this - hope this helps.

**Comment by PaulWay** [comment]
I don't know much about Flask and SQLAlchemy, but maybe this reduces the need for explicit getter and setter methods in the `HostWrapper` object?

**Comment by jhjaggars** [comment]
I'm changing the properties to be underscores.

**Comment by dehort** [comment]
@PaulWay I have merged #44 into master.  Thanks!

The HostWrapper object is just a util class for the test_api tests.  I kept getting the field names wrong so I tried moving the field name "logic" into a class.  With that, the vm helps me make sure that the field names are right.  I'm not completely happy with that solution.  I've looked into replacing that object with a namedtuple but that is immutable so it doesn't work like I want.
